responsibleai/responsibleai/managers/causal_manager.py

```diff
@@ -91,7 +91,12 @@ def add(
         :type treatment_features: list
         :param heterogeneity_features: Features that mediate the causal effect.
         :type heterogeneity_features: list
-        :param nuisance_model: Model type to use for nuisance estimation.
+        :param nuisance_model: Model type to use for nuisance estimation .
+            It is one of {'linear', 'automl'}, optional (default='linear').
```

Comment:
Could you put in a sentence as to what 'nuisance' means for a causal model? I know that this isn't the place to put an in-depth discussion, but this only means something to someone who already knows about the subject area.

Reply:
@riedgar-ms, I have updated the description for nuisance_model and hetrogenity_model. Can you take a look and approve?